apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -41,13 +44,14 @@ const Icon = () => {
   const { themeStatus } = useLocalThemePreference();
 
   if (themeStatus === 'dark') {
-    return <Moon {...headerIconsSettings} />;
+    // FIXME: Revert these changes before merging!
+    return <IconDarkMode size="18" color="blue" />;
```

Comment:
These will be removed before merge! Testing that things work in `web` -- I give it a solid 7/10.

- Icons imported with proper names
- They have the proper defaults (color and size)
- Providing properties changes appearance... but `size` isn't being typed properly (it's using `react-icons`' underlying types